test/helpers/telepresence.go

```diff
@@ -28,7 +29,11 @@ func SetupTelepresence(ctx context.Context) (func(), error) {
 	}
 
 	out, err := exec.CommandContext(ctx, telepresenceExecutable, "helm", "install").CombinedOutput()
-	if err != nil {
+	if err != nil && strings.Contains(string(out), "use 'telepresence helm upgrade' instead to replace it") {
```

Comment:
```suggestion
	if err != nil && bytes.Contains(out, "use 'telepresence helm upgrade' instead to replace it") {
```
and one conversion less

Reply:
Well, we still need to convert the string to [] byte.

Reply:
yo're right heh